### ISSUE TITLE
Update authors

### ISSUE THREAD

**Issue by rgieseke** [hgrecco/pint#694]
Trying to understand the new cool Pandas feature i spotted this:

```
    # I'm happy with both of these as long as Andrew and Zebedee are added on
    # but need to check with Pint Authors...
```
https://github.com/hgrecco/pint/blob/master/pint/pandas_interface/pint_array.py#L6

This should probably be removed and @znicholls and @andrewgsavage be added to AUTHORS?

**Comment by znicholls** [comment]
oops yes should have taken that out, will be fixed as part of resolving #693

**Comment by hgrecco** [comment]
yes please issue a PR. with both changes.
